packages/terra-demographics-banner/src/DemographicsBannerDisplay.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ResponsiveElement from 'terra-responsive-element';
+import 'terra-base/lib/baseStyles';
+
+import './DemographicsBanner.scss';
+import SmallDemographicsBannerDisplay from './_SmallDemographicsBannerDisplay';
```

Comment:
Has this been decided in terms of conventions?  I'm not opposed to it, but if this is the direction we're going I'd like to document it.  I think there are other components being worked on that could benefit from this strategy as well.  I'll add this to the war room.